wemake_python_styleguide/visitors/ast/functions.py

```diff
@@ -149,6 +150,22 @@ def _check_unused_variables(self, node: AnyFunctionDef) -> None:
                 )
         self._check_used_variables(local_variables)
 
+    def _check_argument_default_values(self, node: AnyFunctionDef) -> None:
+
+        for arg in node.args.defaults:
+            if not isinstance(arg, (
```

Comment:
please, move this tuple to the class level, the same as here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/visitors/ast/statements.py#L66-L71